src/azure-cli/azure/cli/command_modules/role/_help.py

```diff
@@ -773,7 +773,13 @@
 helps['role assignment list'] = """
 type: command
 short-summary: List role assignments.
-long-summary: By default, only assignments scoped to subscription will be displayed. To view assignments scoped by resource or group, use `--all`.
+long-summary: >-
+    By default, only assignments scoped to subscription will be displayed. To view assignments scoped by resource or group, use `--all`.
+
+
+    [WARNING] Classic administrators will be deprecated on August 31, 2024.
+    After August 31, 2024, all classic administrators risk losing access to the subscription.
+    Delete Classic Admins who no longer need access or assign an Azure RBAC role for fine-grained access control.
```

Comment:
The wording is inconsistent: the full name `administrators` should be used, instead of `admins`.

```suggestion
    Delete classic administrators who no longer need access or assign an Azure RBAC role for fine-grained access control.
```

Reply:
As discussed, we should use the official name `classic administrators`.